packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Live/LiveNow.tsx

```diff
@@ -104,27 +104,24 @@ export const LiveNow = () => {
       <Collapsible title="Preferences" defaultOpen>
         <List>
           {isOwner && (
-            <ListAction
-              onClick={() => onChatEnabledChange(!chatEnabled)}
-              justify="space-between"
-            >
+            <ListAction justify="space-between" onClick={onChatEnabledToggle}>
```

Comment:
It's best to use the longer form, so the event isn't passed.

```suggestion
            <ListAction justify="space-between" onClick={() => onChatEnabledToggle()}>
```

CC @christianalfoni @CompuIves to verify this.
I know I've been using the long form because of a bug one of you guys reported to me on a PR